rust/headless-client/Cargo.toml

```
@@ -25,6 +25,8 @@ firezone-cli-utils = { workspace = true }
 futures = "0.3.30"
 nix = { version =  "0.28.0", features = ["user"] }
 resolv-conf = "0.7.0"
+# This is a pure Rust re-implementation, so it isn't vulnerable to CVE-2024-3094
+sd-notify = "0.4.1"
```

```suggestion
sd-notify = "0.4.1" # This is a pure Rust re-implementation, so it isn't vulnerable to CVE-2024-3094
```

Nit: Makes sorting these easier.